packages/components/package.json

```diff
@@ -330,6 +330,6 @@
     "dist"
   ],
   "engines": {
-    "node": "16.* || >= 18"
+    "node": ">= 18"
```

Comment:
@alex-ju I'm curious your opinion on this. Technically this should be considered a breaking change, but AFAICT we don't have any known consumers using Node 16. On the other hand, there is no real _need_ for us to make this change either.

Reply:
According to the latest findings around the API our consumers are mostly concerned about the changes not affecting them, rather than following a rigid versioning policy. Based on this, and considering Ember dropped support for Node 16 in v5.11.0, so I'd say we can do it in a minor release too.